src/k8s/pkg/k8sd/api/kubernetes_auth_tokens.go

```diff
@@ -8,30 +8,14 @@ import (
 	"fmt"
 	"net/http"
 
-	apiv1 "github.com/canonical/k8s/api/v1"
+	apiv1 "github.com/canonical/k8s-snap-api/api/v1"
 	"github.com/canonical/k8s/pkg/k8sd/database"
 	databaseutil "github.com/canonical/k8s/pkg/k8sd/database/util"
 	"github.com/canonical/k8s/pkg/utils"
 	"github.com/canonical/lxd/lxd/response"
 	"github.com/canonical/microcluster/v2/state"
 )
 
-func (e *Endpoints) getKubernetesAuthTokens(s state.State, r *http.Request) response.Response {
-	token := r.Header.Get("token")
-
-	var username string
-	var groups []string
-	if err := s.Database().Transaction(r.Context(), func(ctx context.Context, tx *sql.Tx) error {
-		var err error
-		username, groups, err = database.CheckToken(ctx, tx, token)
-		return err
-	}); err != nil {
-		return response.NotFound(err)
-	}
-
-	return response.SyncResponse(true, apiv1.CheckKubernetesAuthTokenResponse{Username: username, Groups: groups})
-}
-
```

Comment:
I assume we are dropping token auth hook here, just making sure.

Reply:
this is just a testing endpoint, it is not used anywhere. the token review webhook is still there and works